app/src/main/java/fr/paug/androidmakers/ui/adapter/DaySchedule.kt

```diff
@@ -1,6 +1,7 @@
 package fr.paug.androidmakers.ui.adapter
 
 data class DaySchedule(
-        val title: String = "",
-        var roomSchedules: MutableList<RoomSchedule> = arrayListOf()
-)
\ No newline at end of file
+  val dayInMillis: Long,
```

Comment:
Super nit: Ikind of like ultra long descriptive names for those like `dayStartEpochMilli`. 
Actually, I'd like to rename a lot of the `time`, `timestamp`, etc.. we have to `fooEpochMilli` everywhere to make it more explicit what the reference point and unit is. But `dayInMillis` works too so feel free to ignore my ramblings!

Reply:
fixed